build.gradle

```diff
@@ -12,9 +12,9 @@ plugins {
 }
 
 repositories {
-  jcenter()
+  mavenLocal()
   google()
-  maven { url "https://oss.sonatype.org/content/repositories/snapshots" } // For Spoon snapshot, until 2.0.0 is released
+  jcenter()
```

Comment:
We need `maven { url "https://oss.sonatype.org/content/repositories/snapshots" }` for Spoon 2.0.